README.md

```diff
@@ -138,86 +153,53 @@ Dependencies are automatically kept up-to-date using [greenkeeper](http://greenk
-| npm test-coverage     | Runs the tests with coverage (watches for changes) |
-| npm test              | Runs the tests once                                |
-| npm lint              | Runs eslint                                        |
-| npm test-coverage-once| Runs the tests once with coverage                  |
-| npm start             | Builds the lib and watches for changes             |
-| npm [run] build       | Builds the lib (used by Travis)                    |
+| Script                  | Description                                        |
+|-------------------------|----------------------------------------------------|
+| npm test                | Runs the tests (watches for changes)               |
+| npm test-coverage       | Runs the tests with coverage (watches for changes) |
+| npm test-once           | Runs the tests once                                |
+| npm lint                | Runs eslint                                        |
+| npm test-coverage-once  | Runs the tests once with coverage                  |
+| test-integration-linter | Runs our integration test-suite                    |
+| npm [run] build         | Builds the lib (used by Travis)                    |
 
-### Building and watching for changes
+### Building
 
-You can run `npm start` to build the library and then rebuild on file changes.
+You can run `npm build` to build the library.
 
 Once you build the library you can use the CLI in `bin/addons-linter`.
 
 ### Testing
 
-Run `npm test`.
+Run `npm test`. This will watch for file-changes and re-runs the test suite.
 
 #### Coverage
 
 We're looking to maintain coverage at 100%. Use the coverage data in the
 test output to work out what lines aren't covered and ensure they're covered.
 
-#### Testing and promises
-
-Tests using promises should return the promise. This removes the need to call
-`done()` in your tests:
-
-```javascript
-it('should do something promise-y', () => {
-  return somePromiseCall()
-    .then(() => {
-      // Assert stuff here.
-    });
-})
-```
-
-To test for rejection you can use this pattern:
-
-```javascript
-import { unexpectedSuccess } from './helpers';
-...
-it('should reject because of x', () => {
-  return somePromiseCall()
-    .then(unexpectedSuccess)
-    .catch((err) => {
-      // make assertions about err here.
-    });
-})
-```
-
 #### Assertions and testing APIs
 
-`assert`, `describe`, `it`, `beforeEach`, and `afterEach` are
-available in tests by default–you don't need to import anything
-for those to work.
+We are using using sinon for assertions, mocks, stubs and more [see the Sinon docs for the API
+available](http://sinonjs.org/).
 
-We're using chai for assertions [see the Chai docs for the API
-available](http://chaijs.com/api/assert/)
+[Jest](https://facebook.github.io/jest/) is being used as a test-runner but also provides helpful tools. Please make sure you read their documentation for more details.
```

Comment:
It might be more useful as a replacement to the chai references to point to the `expect` docs for Jest.